administration/logging.md

```diff
@@ -88,7 +95,7 @@ Following example sets the logging for the Z-Wave binding to **DEBUG**
 log:set DEBUG org.openhab.binding.zwave
 ```
 
-Note that the log levels set using the `log:set` commands are not persistent and will be lost upon restart. To configure those in a persistent way, the commands have to be added to the [configuration file](#config-file).
+Note that the log levels set using the `log:set` commands are persistent and will be applied upon restart.  To modify the stored log levels, use the console or edit the [configuration file](#config-file).
```

Comment:
Please split up to one line per Sentence.

Reply:
Done 👍